### PR TITLE
Typings for expo-localization

### PR DESCRIPTION
**Pull request by burtek**
Repo: https://github.com/expo/expo/tree/master/packages/expo-localization
Docs: https://docs.expo.io/versions/v31.0.0/sdk/localization

Please fill in this template.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/expo-localization/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    /**
+     * List of all the native languages provided by the user settings. These are returned in the order the user defines in their native settings
+     */
+    locales: string[];
+
+    /**
+     * Country code for your device
+     */
+    country: string | undefined;
```
Comment (armanio123) [inline types/expo-localization/index.d.ts]: Based on the documentation this property is a Flow `maybe` type. Meaning it's `string | undefined | null`

File: types/expo-localization/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    /**
+     * A list of all the supported ISO codes
+     */
+    isoCurrencyCodes: string[] | undefined;
```
Comment (armanio123) [inline types/expo-localization/index.d.ts]: Based on the documentation this property is a Flow `maybe` type. Meaning it's `string | undefined | null`

File: types/expo-localization/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    /**
+     * List of all the native languages provided by the user settings. These are returned in the order the user defines in their native settings
+     */
+    const locales: string[];
+
+    /**
+     * Country code for your device
+     */
+    const country: string | undefined;
```
Comment (armanio123) [inline types/expo-localization/index.d.ts]: Based on the documentation this property is a Flow maybe type. Meaning it's string | undefined | null

File: types/expo-localization/index.d.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+    /**
+     * A list of all the supported ISO codes
+     */
+    const isoCurrencyCodes: string[] | undefined;
```
Comment (armanio123) [inline types/expo-localization/index.d.ts]: Based on the documentation this property is a Flow maybe type. Meaning it's string | undefined | null

**Review by armanio123 — CHANGES_REQUESTED** [review]

**Comment by burtek** [comment]
@armanio123 true, missed that one.

**Review by armanio123 — APPROVED** [review]

**Pull request merged**